### ISSUE TITLE
var/sessions must be a symbolic link

### ISSUE THREAD

**Issue by xavier-rdo** [manala/manalize#34]
ansible/group_vars/app.yml (**files_attributes**) should contain : 

``` diff
    - path:  "{{ app.dir }}{{ app.dir_release }}/var/sessions"
      src:   "{{ app.sessions_dir }}"
      state: link_directory
```

Ref.: https://github.com/Elao/symfony-standard/blob/master/ansible/group_vars/app.yml#L50

Good luck ! ;)

**Comment by xavier-rdo** [comment]
Beware of the .gitignore differences btw Symfony standard edition & Elao's standard edition as far as _var_ directory is concerned : 
- https://github.com/symfony/symfony-standard/blob/master/.gitignore#L5
- https://github.com/Elao/symfony-standard/blob/master/.gitignore#L4

**Comment by chalasr** [comment]
Thank you @xavier-rdo! 
We need to update gitignore. First intrusion into the consumer project's, hopefully the last! I'll PR this soon

**Comment by chalasr** [comment]
Fixed in #69
